lte/gateway/c/core/oai/tasks/sgw/s11_causes.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the OpenAirInterface (OAI) Software Alliance under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The OpenAirInterface Software Alliance licenses this file to You under
+ * the terms found in the LICENSE file in the root of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *-------------------------------------------------------------------------------
+ * For more information about the OpenAirInterface (OAI) Software Alliance:
+ *      contact@openairinterface.org
+ */
+
+/*! \file s11_causes.cpp
```

Comment:
Can you use the git mv command for this file?

Reply:
git mv cmd is only used but more than 90% lines in this file are modified. That`s the reason I guess git is not able to identify this file as renamed like other files in this PR

Reply:
if I use the mv command git detects it.

<img width="1135" alt="Screenshot 2022-07-15 at 12 18 42" src="https://user-images.githubusercontent.com/6066106/179204741-9c87cf6d-a879-4a59-89cc-18ebd225d114.png">

Reply:
On the console even for me it shows as renamed with git mv. But once I modify the contents it shows as deleted and added. I did this exercise separately only for this file by renaming the file(using git mv) and then modifying the contents in multiple commits here : https://github.com/magma/magma/compare/master...pruthvihebbani:magma:test_sgw_cpp_migration_s11_causes_file. But once the file modification reaches ~90% git shows it as deleted and added.

Reply:
Ah, I understand. Is the result the same when you rename the file in the last step (commit)?

Reply:
Yes the behavior is same.

Reply:
ok